src/Docker/DevBuilder.php

```diff
@@ -135,7 +142,7 @@ public function build(Repository $config): array
             ServiceFactory::SERVICE_NGINX,
             $config->get(self::NGINX_VERSION, self::DEFAULT_NGINX_VERSION),
             [
-                'ports' => ['443:443'],
+                'ports' => ['8080:8080'],
```

Comment:
Why do we need to expose `8080`?

Reply:
Let me double check if this is necessary...I thought I needed it since it was the port for TLS -> nginx but I think you're right, since it's internal there's no need for it.